classes/locations.php

```diff
@@ -623,9 +628,14 @@ function dragDropUpdateAddress() {
 		function save_post_meta( $post ) {
 			// This function saves the geo data as well as the additional info.
 
-			global $simple_map, $current_screen;
+        if ( ! isset( $_POST['sm_save_postmeta_nonce_field'] ) || ! wp_verify_nonce( $_POST['sm_save_postmeta_nonce_field'], 'sm_save_postmeta_nonce_action' )){
```

Comment:
__[code_sniffer]__
* Line indented incorrectly; expected 12 spaces, found 8
* Expected 0 spaces after opening bracket; 1 found
* Space after opening parenthesis of function call prohibited
* Expected 0 spaces before closing bracket; 1 found
* Expected 1 space after closing parenthesis; found 0
* Line exceeds 120 characters; contains 160 characters

<!-- Comment by [SideCI](https://sideci.com) -->